src/components/organisms/EventModal/EventModal.tsx

```diff
@@ -68,15 +70,17 @@ export const EventModal: React.FC<EventModalProps> = ({
   return (
     <Modal show={show} onHide={onHide} className="EventModal">
       <div className="EventModal__content">
-        <h4 className="EventModal__title">{event.name}</h4>
+        <h4 className="EventModal__title">
+          <RenderMarkdown text={event.name} />
+        </h4>
         <span className="EventModal__subtitle">
           by {event.host} in{" "}
           <button className="button--a" onClick={goToEventLocation}>
             {eventLocationToDisplay}
           </button>
         </span>
 
-        <p className="EventModal__description">{event.description}</p>
```

Comment:
I assume this was done to avoid the ‘p in p’ error, but I don’t think the solution here is what we want to be doing. Without looking too deeply, I would assume that changing the p to a dig and nesting RenderMarkdown within that would have been the needed solution.